services/src/main/java/org/fao/geonet/guiservices/util/GetCsvFileName.java

```diff
@@ -0,0 +1,20 @@
+package org.fao.geonet.guiservices.util;
+
+import jeeves.interfaces.Service;
+import jeeves.server.ServiceConfig;
+import jeeves.server.context.ServiceContext;
+import org.jdom.Element;
+
+import java.nio.file.Path;
+
+public class GetCsvFileName implements Service {
+    @Override
+    public void init(Path appPath, ServiceConfig params) throws Exception {
+
+    }
+
+    @Override
+    public Element exec(Element params, ServiceContext context) throws Exception {
+        return new Element("a").setText("metadata_{datetime}.csv");
```

Comment:
I think the filename should be a configuration to this class so that users can choose their own file format by changing the config-service-csv.xml if they want to.